src/main/java/gr/cytech/chatreminderbot/rest/controlCases/Control.java

```diff
@@ -46,6 +49,8 @@
     private String keyWord_ShowReminders = "list";
 
     private String keyWord_deleteReminder = "delete";
+
+    private String keyWord_ShowSettings = "settings";
```

Comment:
Perhaps now is a good time to fix this variable name and the other ones like it to match Java conventions.
Also, why is it an instance field and not a constant?